### ISSUE TITLE
Purpose of periodic container restarts in NostalgiaForInfinity

### ISSUE THREAD

**Issue by seventhsite** [iterativv/NostalgiaForInfinity#448]
Hi there! 👋

I noticed in the docker-compose.yaml file for the NostalgiaForInfinity strategy that there's a service set up to restart the main container every 86400 seconds. Could you explain the reason behind this setup? Is it for stability, reconnecting to the exchange, or something else? Just curious to understand the thought process here.

Thanks in advance!

**Comment by iterativv** [comment]
Hi, yes, You don't have to use these exact. Just what are your preferences.

**Comment by seventhsite** [comment]
Got it, thanks! So just to clarify, is this included mainly as a suggestion for stability or other reasons? Or is it more of a placeholder for users to customize as needed?

**Comment by iterativv** [comment]
> Got it, thanks! So just to clarify, is this included mainly as a suggestion for stability or other reasons? Or is it more of a placeholder for users to customize as needed?

Yes, in fact you don't have to use this. I don't use docker at all, for example, just native Python to run the bots. It's not required, just in order to update NFI either reload or - yes - some times a complete restart.
